migratedb.d/05.sql

```diff
@@ -0,0 +1,23 @@
+
+DO
+$$
+DECLARE
+-- The version we know how to do migration from, at the end of a successful migration
+-- we will no longer be at this version.
+  sourcever INTEGER := 4;
+  changes VARCHAR := 'Add field for correlation ids';
+BEGIN
+-- No explicit transaction handling here, this all happens in a transaction
+-- automatically
+  IF (select max(version) from local_ega.dbschema_version) = sourcever then
```

Comment:
```suggestion
  IF (select max(version) from sda.dbschema_version) = sourcever then
```

I believe this would be after the `local_ega`  to `sda` switch?

Reply:
I think it does not matter but since it has been change above it makes sense to change as well here